src/domain/submission/services/exporter/jwt.ts

```diff
@@ -0,0 +1,6 @@
+import { sign } from 'jsonwebtoken';
+
+// MECA workaround for issue https://github.com/libero/reviewer/issues/1165
+export const encode = (secret: string, payload: object, expiresIn: string): string => {
+    return sign(payload, secret, { expiresIn, issuer: 'continuum-adaptor' });
```

Comment:
should the issuer different?

Reply:
Would be good to keep it the same to avoid any unexpected failures, it's there to replicate a genuine token. This is not the ideal solution and is a work around.

Reply:
I'll approve for now but we probably should address this later